table.c

```diff
@@ -0,0 +1,117 @@
+#define _CRT_SECURE_NO_WARNINGS
+#define HASH_TABLE 100
```

Comment:
```suggestion
#define HASH_TABLE_SIZE 100
```
Надо, чтобы имя переменной отражало её назначение.